chia/util/cached_bls.py

```diff
@@ -0,0 +1,49 @@
+import functools
+import hashlib
+from typing import List
+
+from blspy import AugSchemeMPL, G1Element, G2Element, GTElement
+from chia.util.lru_cache import LRUCache
+
+
+def get_pairings(cache: LRUCache, pks: List[G1Element], msgs: List[bytes], force: bool) -> List[GTElement]:
+    pairings: List[GTElement] = []
+    missing_count: int = 0
+    for pk, msg in zip(pks, msgs):
+        aug_msg = bytes(pk) + msg
+        h = hashlib.sha256(aug_msg).digest()
```

Comment:
We should use std_hash here instead of hashlib

Reply:
Fixed